### ISSUE TITLE
Cannot change custom.css

### ISSUE THREAD

**Issue by tepiloxtl** [filebrowser/filebrowser#1764]
After making changes to custom.css and refreshing the branding settings, nothing changes. File that is linked as "/static/custom.css" in the webpage source doesn't reflect changes made to the file in the location specified, even after serving an empty custom.css and after restarting filebrowser

**Comment by topa-LE** [comment]
I also have the same problem. Even a new custom.css and also the custom logo.svg in the /img folder is not defined.

About the global settings in the backend with path to CSS
/mnt/user/filebrowser/config

no custom.css is loaded!

**Comment by tepiloxtl** [comment]
It seems to be due to web browser caching the files and not letting them go when they change on the server side. Not sure how it should be dealt with, but thats my observation

**Comment by topa-LE** [comment]
Is there any explanation or hint why a custom.css and img do not work although a path on the server was entered in the Global Settings?

(Example : /mnt/filebrowser/custom/)

There is the custom.css with your own code.

also a folder /img with your logo.svg

**Comment by dacagi** [comment]
I struggled with this for quite a bit until I found what I consider a workaround.

I was trying to pass a _branding.files_ variable to point it to the folder where I placed the _custom.css_. No images in my case, I'm don't know about those.. According to the _Branding_ struct in the code it allows for the following:

```
// Branding contains the branding settings of the app.
type Branding struct {
	Name            string `json:"name"`
	DisableExternal bool   `json:"disableExternal"`
	Files           string `json:"files"`
	Theme           string `json:"theme"`
	Color           string `json:"color"`
}
```
I have been unable to pass this by using a config file like with the rest of the _Server_ settings, it seems these _Settings_ are handled differently.

My solution was to just let the app load the first time with the basic config, and then update the path to branding.files inside the web UI in the settings menu.
